storage/storage-client/src/lib.rs

```diff
@@ -173,6 +173,10 @@ impl DbReader for StorageClient {
     fn get_epoch_ending_ledger_info(&self, _: u64) -> Result<LedgerInfoWithSignatures> {
         unimplemented!()
     }
+
+    fn get_block_timestamp(&self, _version: u64) -> Result<u64> {
```

Comment:
what else in the block metadata we are potentially exposing? like the `round`, the `proposer`, etc? @metajack , any ideas?

I think we'd better expose them all at the storage level and probably trim the size down at the json-rpc level.

Reply:
at least for JSON RPC, currently there are no plans to expose other block info (e.g. `round`, `proposer`) in block metadata, but other use cases (not just JSON RPC) might come up in the future?

Reply:
I know what you are thinking, you want to implement an get_preceeding_blockmetadata_txn  or even get_blockmetadata_txn_version()  ?  Right now we don't really have that need